roq-frontmatter/runtime/src/main/java/io/quarkiverse/roq/frontmatter/runtime/model/PageInfo.java

```diff
@@ -28,6 +28,11 @@ public record PageInfo(
          */
         String dateString,
 
+        /**
+         * The markup (markdown, asciidoc) or null if none
+         */
+        String markup,
```

Comment:
What is this used for ?

Reply:
(It was added to deal with the notitle, but then I thought it was cleaner/nicer to have attributes in the layout)